cmd/commands/runtime.go

```diff
@@ -1554,6 +1568,21 @@ func postInstallationHandler(ctx context.Context, opts *RuntimeInstallOptions, e
 	printSummaryToUser()
 }
 
+func HandleCliStep(ctx context.Context, r reporter.AnalyticsReporter, event reporter.CliEventType, message string, err error) {
+	status := reporter.SUCCESS
+	appendLogToSummary(message, err)
```

Comment:
this should not sit between the `status` declaration and the `if`. maybe move it to the beginning of this function.